src/nvim/move.c

```diff
@@ -74,8 +71,9 @@ static void comp_botline(win_T *wp)
   }
 
   for (; lnum <= wp->w_buffer->b_ml.ml_line_count; ++lnum) {
-    last = lnum;
-    folded = FALSE;
+    int n;
+    linenr_T last = lnum;
+    int folded = FALSE;
```

Comment:
OK. I let this in suspens. I will do a patch that remove all FALSE / TRUE of this file.